src/content/docs/ru/tutorial/2-pages/2.mdx

```diff
@@ -0,0 +1,197 @@
+      </head>
+      <body>
+        <a href="/">Главная</a>
+        <a href="/about/">Обо мне</a>
+        <a href="/blog/">Блог</a>
+
+        <h1>Мой блог об изучении Astro</h1>
+        <p>Здесь я буду писать о своем путешествии в изучении Astro.</p>
+        <ul>
+          <li><a href="/posts/post-1/">Пост 1</a></li>
+        </ul>
+      </body>
+    </html>
+    ```
+
+2. Теперь добавьте еще два файла в папке `src/pages/posts/`: `post-2.md` и `post-3.md`. Вот небольшой пример кода, который вы можете скопировать и вставить в эти файлы, или написать свой:
+
+    ```md title="src/pages/posts/post-2.md"
+    ---
+    title: Мой второй пост в блоге
+    author: Ученик Astro
+    description: "После того, как я изучил немного Astro, я не мог остановиться!"
+    image:
+        url: "https://docs.astro.build/assets/arc.webp"
+        alt: "Привью Astro c радугой-дугой"
+    pubDate: 2022-07-08
+    tags: ["astro", "блогинг", "обучение", "успех"]
+    ---
+    После успешной первой недели изучения Astro, я решил попробовать еще. Я создал небольшой компонент из памяти и импортировал его!
+    ```
+
+    ```md title="src/pages/posts/post-3.md"
+    ---
+    title: Мой третий пост в блоге
+    author: Ученик Astro
+    description: "У меня были некоторые трудности, но помощь сообщества была бесценной!"
+    image:
+        url: "https://docs.astro.build/assets/rays.webp"
+        alt: "Превью изображение из лучей Astro."
+    pubDate: 2022-07-15
+    tags: ["astro", "блогинг", "обучение", "тяжко", "сообщество"]
+    ---
+    Не всегда было легко, но я наслаждаюсь работой с Astro. А [сообщество в Discord](https://astro.build/chat) действительно дружелюбное и полезное!
+    ```
+
+3. Добавьте ссылки на эти посты:
+
+    ```astro title="src/pages/blog.astro" ins={18-19}
+    ---
+    ---
+    <html lang="ru">
+      <head>
+        <meta charset="utf-8"/>
+        <meta name="viewport" content="width=device-width" />
+        <title>Astro</title>
+      </head>
+      <body>
+        <a href="/">Главная</a>
+        <a href="/about/">Обо мне</a>
+        <a href="/blog/">Блог</a>
+
+        <h1>Мой блог об изучении Astro</h1>
+        <p>Здесь я буду писать о своем путешествии в изучении Astro.</p>
+        <ul>
+          <li><a href="/posts/post-1/">Пост 1</a></li>
+          <li><a href="/posts/post-2/">Пост 2</a></li>
+          <li><a href="/posts/post-3/">Пост 3</a></li>
+        </ul>
+      </body>
+    </html>
+    ```
+
+4. Проверьте предварительный просмотр в вашем браузере и удостоверьтесь, что:
+
+   - Все ваши ссылки на Post 1, Post 2 и Post 3 ведут на рабочую страницу на вашем сайте. (Если вы обнаружили ошибку, проверьте ваши ссылки на `blog.astro` или имена файлов Markdown.)
+
+<Box icon="question-mark">
+### Проверьте свои знания
+
+1. Контент в файле Markdown (`.md`) преобразуется в:
+  <MultipleChoice>
+    <Option isCorrect>HTML</Option>
+    <Option>CSS</Option>
+    <Option>JavaScript</Option>
+  </MultipleChoice>
```

Comment:
```suggestion
    <MultipleChoice>
        <Option isCorrect>HTML</Option>
        <Option>CSS</Option>
        <Option>JavaScript</Option>
    </MultipleChoice>
```
The `<MultipleChoice>` component needs to be indented when inside an ordered list to render correctly